### PR TITLE
[Doc] Add content for topic-level schema compatibility check strategy

### PR DESCRIPTION
**Pull request by momo-jun**
Doc updates for https://github.com/apache/pulsar/pull/13297.

### Modifications
1. Simplify the description of strategy priority set at different levels.
image

2. Add set/get/remove methods for topic-level schema compatibility check strategy in three ways.
2.1) admin CLI
image

2.2) Rest API
image

  2.3) Java Admin API
image
image

### Documentation
- [x] `doc`

### CODE REVIEW

**Comment by momo-jun** [comment]
Hi @nodece @codelipenghui @Anonymitaet Can you pls review this doc PR? Thanks.

File: site2/docs/schema-manage.md [hunk 1]
```diff
@@ -809,17 +809,118 @@ To use your custom schema storage implementation, perform the following steps.
...
 
-### Namespace 
+
+### Topic level
+
+To set a schema compatibility check strategy at the topic level, use one of the following methods.
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--admin CLI-->
```
Comment (nodece) [inline site2/docs/schema-manage.md:829]:
```suggestion
<!--Admin CLI-->
```

File: site2/docs/schema-manage.md [hunk 2]
```diff
@@ -809,17 +809,118 @@ To use your custom schema storage implementation, perform the following steps.
...
+The schema compatibility check strategy set at different levels has priority: topic level > namespace level > broker level. 
 
-- If you do not set schema compatibility check strategy at both namespace or broker level, it uses the `FULL` strategy.
+- If you set the strategy at both topic and namespace level, it uses the topic-level strategy. 
 
-- If you set schema compatibility check strategy at broker level rather than namespace level, it uses the strategy set for the broker level.
+- If you set the strategy at both namespace and broker level, it uses the namespace-level strategy.
 
-- If you set schema compatibility check strategy at namespace level rather than broker level, it uses the strategy set for the namespace level.
+- If you do not set the strategy at any level, it uses the `FULL` strategy.
```
Comment (nodece) [inline site2/docs/schema-manage.md:821]: Could you add a description that available values for schema compatibility strategy?

File: site2/docs/schema-manage.md [hunk 3]
```diff
@@ -809,17 +809,118 @@ To use your custom schema storage implementation, perform the following steps.
...
+admin.topicPolicies().setSchemaCompatibilityStrategy("my-tenant/my-ns/my-topic", SchemaCompatibilityStrategy.ALWAYS_INCOMPATIBLE);
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+<br />
+To get the topic-level schema compatibility check strategy, use one of the following methods.
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--admin CLI-->
```
Comment (nodece) [inline site2/docs/schema-manage.md:860]:
```suggestion
<!--Admin CLI-->
```

File: site2/docs/schema-manage.md [hunk 4]
```diff
@@ -809,17 +809,118 @@ To use your custom schema storage implementation, perform the following steps.
...
+admin.topicPolicies().getSchemaCompatibilityStrategy("my-tenant/my-ns/my-topic", false);
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+<br />
+To remove the topic-level schema compatibility check strategy, use one of the following methods.
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--admin CLI-->
```
Comment (nodece) [inline site2/docs/schema-manage.md:895]:
```suggestion
<!--Admin CLI-->
```

File: site2/docs/schema-manage.md [hunk 5]
```diff
@@ -809,17 +809,118 @@ To use your custom schema storage implementation, perform the following steps.
...
+```java
+PulsarAdmin admin = …;
+
+admin.removeSchemaCompatibilityStrategy("my-tenant/my-ns/my-topic");
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+
+### Namespace level
```
Comment (nodece) [inline site2/docs/schema-manage.md:923]: We should keep the `DOCUSAURUS_CODE_TABS` tab has same style between Topice level and Namespace level.

Comment (momo-jun) [reply site2/docs/schema-manage.md:923]: > We should keep the `DOCUSAURUS_CODE_TABS` tab has same style between Topice level and Namespace level.

Do you mean by using the same tab name, like Admin CLI?

Comment (nodece) [reply site2/docs/schema-manage.md:923]: Yes.

File: site2/docs/schema-manage.md (hunk 2 above)
Comment (momo-jun) [reply site2/docs/schema-manage.md:821]: Good point. I will add a link as the reference for available values. Thank you.

File: site2/docs/schema-manage.md (hunk 1 above)
Comment (momo-jun) [reply site2/docs/schema-manage.md]: Corrected. PTAL. Thanks.

File: site2/docs/schema-manage.md (hunk 3 above)
Comment (momo-jun) [reply site2/docs/schema-manage.md]: Corrected. PTAL. Thanks.

File: site2/docs/schema-manage.md (hunk 4 above)
Comment (momo-jun) [reply site2/docs/schema-manage.md]: Corrected. PTAL. Thanks.

File: site2/docs/schema-manage.md (hunk 5 above)
Comment (momo-jun) [reply site2/docs/schema-manage.md:923]: Corrected. PTAL. Thanks.

**Review by nodece — APPROVED** [review]
LGTM

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**